src/Json.Schema.Validation/RuleResources.resx

```diff
@@ -118,68 +118,68 @@
   </data>
   <data name="ErrorInvalidDependencyType" xml:space="preserve">
-    <value>JSON schema requires the value of each property of the "dependencies"  object, to be either an schema or an array of strings, but a token of type "{1}" was seen.</value>
+    <value>JSON schema requires each property of the 'dependencies' object to be either an schema or an array of strings, but a token of type '{1}' was seen.</value>
   </data>
   <data name="ErrorInvalidEnumValue" xml:space="preserve">
-    <value>The value "{1}" does not match any of the enum values [{2}].</value>
+    <value>'{1}' does not match any of the enum values [{2}].</value>
   </data>
   <data name="ErrorInvalidItemsType" xml:space="preserve">
-    <value>JSON schema requires the value of "items" to be either a schema or an array of schemas, but a token of type "{1}" was seen.</value>
+    <value>JSON schema requires 'items' to be either a schema or an array of schemas, but a token of type '{1}' was seen.</value>
   </data>
   <data name="ErrorInvalidPropertyDependencyType" xml:space="preserve">
-    <value>JSON requires that if the value of a property of the "dependencies" object is an array, then every element of the array must be a string, but a token of type "{1}" was seen.</value>
+    <value>JSON requires that if a property of the 'dependencies' object is an array, then every element of the array must be a string, but a token of type '{1}' was seen.</value>
   </data>
   <data name="ErrorInvalidTypeString" xml:space="preserve">
-    <value>"{1}" is not a valid schema type.</value>
+    <value>'{1}' is not a valid schema type.</value>
   </data>
   <data name="ErrorInvalidTypeType" xml:space="preserve">
-    <value>JSON schema requires the value of "type" to be either a string or an array of strings, but a token of type "{1}" was seen.</value>
+    <value>JSON schema requires 'type' to be either a string or an array of strings, but a token of type '{1}' was seen.</value>
   </data>
   <data name="ErrorMessageStringWithPath" xml:space="preserve">
-    <value>at "{{0}}": {0}</value>
+    <value>{{0}}: {0}</value>
   </data>
   <data name="ErrorNotAllOf" xml:space="preserve">
-    <value>The instance does not validate successfully against all of the {1} schemas specified by "allOf".</value>
+    <value>The instance does not validate successfully against all of the {1} schemas specified by 'allOf'.</value>
   </data>
   <data name="ErrorNotAMultiple" xml:space="preserve">
-    <value>The value {1} is not a multiple of {2}.</value>
+    <value>{1} is not a multiple of {2}.</value>
   </data>
   <data name="ErrorNotAnyOf" xml:space="preserve">
-    <value>The instance does validate successfully against any of the {1} schemas specified by "anyOf".</value>
+    <value>The instance does validate successfully against any of the {1} schemas specified by 'anyOf'.</value>
```

Comment:
>instance [](start = 15, length = 8)

This is an example of a rule that needs to have the path encoded in it. I know we've deferred this work for now.